solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/SolrZkClient.java

```diff
@@ -85,7 +85,7 @@ public class SolrZkClient implements Closeable {
   private int minStateByteLenForCompression;
 
   public MapWriter getMetrics() {
-    return metrics::writeMap;
+    return metrics;
```

Comment:
Not needed in this PR, I assume

Reply:
It was causing linting errors in the CI/CD, but not locally, which is why I had to fix it. Same case for `p::writeMap` in another commit.